GitUI/UserControls/GPGKeys/GpgKeyDisplayInfo.cs

```diff
@@ -0,0 +1,100 @@
+﻿using ResourceManager;
+
+namespace GitUI.UserControls.GPGKeys
+{
+    public class GpgKeyDisplayInfo : IEquatable<GpgKeyDisplayInfo>
```

Comment:
Can this be `struct` or `readonly struct`?

Reply:
I'll look into it.  Probably.

Reply:
Started work on this.